### ISSUE TITLE
Conflicts with current version of arviz/xarray

### ISSUE THREAD

**Issue by gjgilbert** [exoplanet-dev/exoplanet#310]
**Describe the bug**
In a clean conda environment, exoplanet fails to import, raising two dozen or so traceback errors leading through exoplanet --> pymc3 --> arviz --> xarray. 

Forcing xarray==2022.9.0 fixes the issue, and also sets arviz==0.12.1

**To Reproduce**
In terminal:
```
conda create --name test1
conda activate test1
python -m pip install -U exoplanet
```
In python:
`import exoplanet
`

**Expected behavior**
Exoplanet should work on pip installation.

Inside the clean conda environment, running

```
python -m pip install -U exoplanet
pip install xarray==2022.9.0
```

seems to fix the issue

**Your setup (please complete the following information):**
 - Version of exoplanet: 0.5.3
 - Operating system: macOS 12.1 (M1 chip)
 - Python version & installation method (pip, conda, etc.): python=3.9, virtual environment created with conda, pip install everything else

**Additional context**
I think the root issue is conflicting numpy requirements, but I haven't investigated any further.

**Comment by dfm** [comment]
Thanks! We've fixed this in a few places, but it hasn't been propagated to exoplanet yet. Over in #309, it's fixed using the following requirements:

```python
"pymc3>=3.9", "numpy<1.22", "xarray<2023.10.0"
```

Would you be willing to open a PR to the main branch adding a constraint like this so that we can release an update sooner?

**Comment by gjgilbert** [comment]
Opened a PR [#311](https://github.com/exoplanet-dev/exoplanet/pull/311). In the future for small issues such as this would it be better to make a PR directly rather than opening an issue?

**Comment by dfm** [comment]
Fixed in #309
